config/locales/doc_auth/fr.yml

```diff
@@ -8,6 +8,7 @@ fr:
     buttons:
       add_new_photos: Ajoutez de nouvelles photos
       continue: Continuer
+      have_phone: Oui j’ai un téléphone
```

Comment:
Nit: Could you add a comma after "Oui"?

Reply:
yeah, i had asked myself the same question 🤔  but after checking how we translate across the site, not using the comma after yes/no is the pattern we use everywhere else